setup_env.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
Nitpick ("best practices"):

```suggestion
#!/usr/bin/env bash
```